docs/markdown/Users.md

```diff
@@ -4,10 +4,11 @@ title: Users
 
 # List of projects using Meson
 
-If you have a project that uses Meson that you want to add to this
-list, please [file a
-pull-request](https://github.com/mesonbuild/meson/edit/master/docs/markdown/Users.md)
-for it. All the software on this list is tested for regressions before
+If you're aware of a notable project that uses Meson, please
+[file a pull-request](https://github.com/mesonbuild/meson/edit/master/docs/markdown/Users.md)
+for it. For other projects using Meson, you may be interested in this
+[GitHub search](https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29meson%5C.build%24%2F&type=code).
+All the software on this list is tested for regressions before
 release, so it's highly recommended that projects add themselves
 here. Some additional projects are listed in the [`meson` GitHub
```

Comment:
AFAIK, this is not true and inviting projects to be listed here to get test coverage is misleading.

Reply:
Agreed, fixed.